pkg/apis/performanceprofile/v1/performanceprofile_types.go

```diff
@@ -98,6 +98,9 @@ type CPU struct {
 	// Defaults to "true"
 	// +optional
 	BalanceIsolated *bool `json:"balanceIsolated,omitempty"`
+	// Offline defines a set of CPUs that will be unused and set offline
+	// +optional
+	Offlined *CPUSet `json:"offlined"`
```

Comment:
Why are we adding the offlined field to v1 ?   do we need to support both v1 and v2 with new additions ? 
(cc @fromanirh)

Reply:
right, good catch. We need to add it to `v2` only actually

Reply:
done